reference/test/solver/upper_trs_kernels.cpp

```diff
@@ -49,26 +49,31 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <ginkgo/core/stop/time.hpp>
 
 
-#include "core/solver/upper_trs_kernels.hpp"
-#include "core/test/utils/assertions.hpp"
+#include <core/solver/upper_trs_kernels.hpp>
```

Comment:
nit: Use `""` instead of `<>` here.

Reply:
I thought mike was going to change all of these with his PR, which doesn't hurt since anyway both versions are implementation defined...